windows/installer.nsi

```diff
@@ -40,7 +40,10 @@ BrandingText "${APP_NAME}. Make music online. With friends. For free."
 
 ; Add support for copying registry keys
 
-!insertmacro COPY_REGISTRY_KEY
+!if ${BUILD_OPTION} != "jackonwindows"
+    ;Only insert macro when not building for JACK
```

Comment:
```suggestion
    ; This macro is only needed when using ASIO4ALL
```

Reply:
Understand the idea, but I used this text to be inline with the != option. So it will use the macro for ASIO4ALL, but also for other buildoptions they may be added in the future.

Reply:
> So it will use the macro for ASIO4ALL, but also for other buildoptions they may be added in the future.

Hrm, yes, I think this is what would happen. However, the intention of the code is to limit the macro inclusion to builds which support ASIO/ASIO4ALL, isn't it?
And sadly we cannot check this explicitly with the current code, which is why we resort to the negative check?

I'm usually in favor of adding comments about the (non-obvious) intention of code rather than comments which repeat (and maybe elaborate on) what the code already says.

I wonder if we should do both here?

```suggestion
    ; This macro is only needed when using ASIO4ALL.
    ; Technically, we cannot detect this easily.
    ; Therefore, we only insert the macro when not building for JACK.
```